spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -41,6 +41,7 @@ object SpdxLicenseAliasMapping {
         "AFLv2.1" to AFL_2_1,
         "agpl" to AGPL_3_0_ONLY,
         "ALv2" to APACHE_2_0,
+        "APACHEv2" to APACHE_2_0,
```

Comment:
Should it go above line 49?

Reply:
I'll sort the whole list as part of another PR.